app/models/api_docs/service.rb

```diff
@@ -93,6 +99,30 @@ def specification
   end
 
   private
+
+  def assign_all_attributes(attributes)
+    assign_service_from_attributes(attributes)
```

Comment:
Please, I would really prefer we stick with ActiveRecord API here. This is unnecessarily overcomplicating IMO.